ruma-events/src/room/member.rs

```diff
@@ -54,6 +54,7 @@ pub struct MemberEventContent {
     pub is_direct: Option<bool>,
 
     /// The membership state of this user.
+    #[ruma_event(skip_redacted)]
```

Comment:
Why is this not producing an error?

```suggestion
    #[ruma_event(skip_redaction)]
```